soc/nordic/nrf54h/Kconfig.soc

```
@@ -6,6 +6,7 @@
 config SOC_NRF54H20
 	bool
 	select SOC_SERIES_NRF54HX
+	select ARM_ON_ENTER_CPU_IDLE_HOOK if PM_S2RAM
```

```suggestion
	select ARM_ON_ENTER_CPU_IDLE_HOOK
```

The hook is not required for S2RAM, but for S2IDLE

---

> The hook is not required for S2RAM, but for S2IDLE

The default configuration ensures us, that for APP the DOMAIN_0 is powered on : https://github.com/nrfconnect/sdk-zephyr/blob/main/soc/nordic/nrf54h/soc.c#L55-L58
For RADIO it is not required as you said - do we assume, that something will change this setting on the fly then? Or the initial setting is invalid and should be changed?